core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -886,8 +886,9 @@ protected String getParentPath(String path) {
    * @return true if the path is the root, false otherwise
```

Comment:
perhaps add some javadoc with examples to this method to more clearly illustrate it's inputs/outputs?